docker-compose.yml

```diff
@@ -616,6 +667,12 @@ services:
       - "6379"
     ports:
       - "${TEST_REDIS_PORT}:6379"
+  redis_old:
+    image: redis:3.0
+    expose:
+      - "6379"
+    ports:
+      - "${TEST_REDIS_OLD_PORT}:6379"
```

Comment:
Since we're adding this just for qless, I suggest leaving a comment here stating so -- why this is needed, and that we can remove it if/when we drop support for qless.